server/i18n/en.json

```diff
@@ -5927,6 +5927,10 @@
     "id": "app.oauth.update_app.updating.app_error",
     "translation": "We encountered an error updating the app."
   },
+  {
+    "id": "app.plugin.blocked.app_error",
+    "translation": "Plugin {{.Id}} cannot be installed since it's already packaged with Mattermost."
```

Comment:
I did make this generic enough that we could use it for other things. Maybe the error message should be generic too?

Reply:
0/5, but the debug log already mentions Skipping installation of plugin since plugin is on blocklist. Do you still want me to change it?

Reply:
Doesn't really matter. If we use it's genetic form we can change the message.

Reply:
Any suggestions on what the message should be? I want to include some reason as well, because just reading that the plugin is blocked will make users ask why.

Reply:
`Plugin {{.Id}} is on the block list. Some plugins are blocked because they are built into this version of Mattermost`